.github/workflows/gh-ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         fail-fast: false
         matrix:
           os: [ubuntu-latest, ]
-          python-version: [3.8, 3.9, "3.10"]
+          python-version: [3.8, 3.9, "3.10", "3.11.0-rc.2"]
```

Comment:
I suspect our non minimal deps will fail here, like rdkit and OpenMM

Might need to put this as a separate entry with a minimal deps argument.